src/h5web/providers/mock/data.ts

```diff
@@ -21,7 +21,7 @@ import {
   makeDatatype,
   makeExternalLink,
   makeGroup,
-  makeNxData,
+  makeNxDataGroup,
```

Comment:
Adding back the `group` suffix for consistency with `findNxDataGroup` (as in the group entity) and `useNxData` (as in the parsed NXdata info).